ui/admin/tests/acceptance/accounts/delete-test.js

```diff
@@ -30,18 +30,19 @@ module('Acceptance | accounts | delete', function (hooks) {
       global: null,
       org: null,
     },
-    authMethods: null,
+    authMethod: null,
```

Comment:
did this not work previously? curious how this didn't fail

Reply:
oh yup it did work before since it was being defined someplace else, mainly just added it to follow pattern of declaring it first.